### PR TITLE
data: OmkarBorhade98 LinkFree Profile

### PR DESCRIPTION
**Pull request by OmkarBorhade98**
## Check List

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Note to reviewers
Added my LinkFree profile

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6641"></a>

### CODE REVIEW

File: data/OmkarBorhade98.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Omkar Borhade",
+  "type": "personal",
+  "bio": "Mechanical Engineer turned into Software Devolaper. Avid Reader"
```
Comment (khushalarora) [inline data/OmkarBorhade98.json:4]: Please add a **'Comma'** at the end of this line :+1: 
```Suggestion
  "bio": "Mechanical Engineer turned into Software Devolaper. Avid Reader",
```

File: data/OmkarBorhade98.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  "links": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/Omkar_Borhade98",
+      "icon": "FaTwitter"
+    }
```
Comment (khushalarora) [inline data/OmkarBorhade98.json:10]: Please add a **'Comma'** here also :+1: 
```Suggestion
    },
```

File: data/OmkarBorhade98.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    {
+      "name": "Github",
+      "url": "https://github.com/OmkarBorhade98",
+      "icon": "FaGithub"
+    }
```
Comment (khushalarora) [inline data/OmkarBorhade98.json:15]: Please add a **'Comma'** here also :+1: 
```Suggestion
    },
```

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hey @OmkarBorhade98, thank you for contributing :tada:

**Comment by OmkarBorhade98** [comment]
Thanks @khushalarora I've made the necessary changes

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**